### ISSUE TITLE
Suggestion: Total Stack Price

### ISSUE THREAD

**Issue by Hiethen** [shirsig/aux-addon#153]
Would it be helpful to add a stack price in the Post screen so that you know the total price of the item you are selling as a stack. For example, I was trying to sell a stack of five and entered 65 silver for unit starting and unit buyout price. When I posted the auction, the total stack starting and buyout price was 3 gold 25 silver.

**Comment by pisikak** [comment]
YES! I need this function too cause if price is to low the unit price post can cause some issues . for example. i need to post 13s99c for stuck of 10items. ok i devide 13,99 / 10 = 1.399 . This means i need to post it dor 1,399 per UNIT . But it impossible cause aux returns the numbers... so the full price for stack will be 13s 00 c.. not 13 s 99 c .. So... WHAT CAN I DO??? i can just go to blizzard ui.. but Why then need this addon? So. if you add FULL STACK PRICE it will be great

**Comment by shirsig** [comment]
You can enter fractions of copper even if you don't see them displayed. The automatic undercutting also uses fractions internally so it always undercuts the stack by 1c.

**Comment by pisikak** [comment]
thx for fast answer! i tryed and it works now. for example if i need 8s99c i made id 44.95 c per unit . 
cool .
